tests/test_agent_runner_repo.py

```
@@ -0,0 +1,76 @@
+"""‏`agent-runner.ps1` עובד על ריפו שנמסר לו, ולא על אחד קבוע.
+
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)